components/common/Text.tsx

```diff
@@ -31,6 +35,21 @@ const LText = ({ children, style, centered, marginHorizontal = 0, size, onPress
     mergedStyle = StyleSheet.compose(mergedStyle, { fontSize: size });
   }
 
+  const scaleAnim = scaleAnimation;
```

Comment:
I would have probably preferred to do this on a separate component named `AnimatedText`